.travis.yml

```diff
@@ -4,8 +4,8 @@ lein: 2.8.1
 jdk:
   - oraclejdk8
   - oraclejdk9
-  - oraclejdk10
-  - oraclejdk11
+  # - openjdk10   # TravisCI deprecated their Oracle JDK 10 image, annoyingly...
```

Comment:
This should not be commented out, since line 20 works around TravisCI's terrible base openjdk10 image.

Reply:
my bad, I reverted the change